Pear/Supporting/Secrets.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Secrets.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 2/28/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import Foundation
+
+struct Secrets {
```

Comment:
It'd be great if we can consolidate this with keys.swift! Same with consolidating `Keys.plist` with `Secrets.plist`.